src/groupByKey.ts

```diff
@@ -2,6 +2,18 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: keyof T) {
```

Comment:
![image](https://github.com/mate-academy/ts_group_by_key/assets/56752562/46352e02-59b4-4ada-b779-525b0f01a967)
Try to fix linter error